NInce_Hw1

NesibeInce opened the pull request.

stasel commented on week1/homework/nodejs-exercises/exercise1/app.js:
```diff
@@ -0,0 +1,6 @@
+numbers = ["12", "846", "2", "1236"]
+
+numbers.forEach(function (number) {
+  console.log(padLeft.padLeft(number, 4, ""));
```
This code is not working properly: check our `padLeft` function again, it receives `str` as a third parameter, what's its use in the function? Perhaps you need to call `padLeft` with something different than empty string (`""`)?

You are also calling `padLeft` twice. call it once should be enough.

NesibeInce replied:
I got your point, thanks for that..

NesibeInce replied:
I implemented the feedback. 
Thank you for the details.

stasel commented on week1/homework/nodejs-exercises/exercise2/app.js:
```diff
@@ -0,0 +1,6 @@
+numbers = ['12', '846', '2', '1236'];
+
+numbers.forEach(function(number) {
+  console.log(leftPad(number, 8, ''));
```
Same comment here like in exercise 1. 
Hint: We need to pad with spaces

NesibeInce replied:
the same as above.

NesibeInce replied:
I implemented it too.
Thank you for your care.

stasel commented on week1/homework/nodejs-exercises/exercise3/app.js:
```diff
@@ -0,0 +1,54 @@
+var http = require('http');
+
+//create a server
+let server = http.createServer(function (req, res) {
```
You can also use the arrow function: `http.createServer((req, res) => {`

NesibeInce replied:
that will be better :)

NesibeInce replied:
Now it looks better!

stasel commented on week1/homework/nodejs-exercises/exercise3/app.js:
```diff
@@ -0,0 +1,54 @@
+      padding: 10px;
+    }
+    `
+    )
```
Good job implementing `style.css` 👍

NesibeInce replied:
thanks for your comments..

stasel commented on week1/homework/nodejs-exercises/exercise3/app.js:
```diff
@@ -0,0 +1,54 @@
+        <script src="script.js"></script>
+        </body>
+    </html>
+  `
```
Very nice formatting of the HTML 👍

NesibeInce replied:
Thanks

stasel commented on week1/homework/nodejs-exercises/exercise1/andrejs-awesome-function.js:
```diff
@@ -0,0 +1,5 @@
+  return '00000'.replace(/0/g, str).slice(0, num - val.length) + val;
+}
+
+exports.padLeft = padLeft;
```
If you change this to `exports = padLeft;` then you don't have to type `padLeft.padLeft` to use it but only `padLeft(number, 4, "")` once

NesibeInce replied:
I understood..

NesibeInce replied:
I implemented all the feedback, thank you for your close attention.

NesibeInce commented:
Thanks @stasel for your effort and feedback, I will implement all and PR it again..

wouterkleijn commented:
Thanks for the extensive feedback @stasel 

I love your communication here @NesibeInce Great job, looking forward to the updated PR :)

wouterkleijn commented:
Hey @NesibeInce Seems like this still needs a final update. Can you have a look?

NesibeInce commented:
Oh, ok!
 I will do it today.
Thank you for following up! 🙂

On Sun, Dec 8, 2019 at 8:36 AM Wouter Kleijn <notifications@github.com>
wrote:

> Hey @NesibeInce <https://github.com/NesibeInce> Seems like this still
> needs a final update. Can you have a look?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HackYourHomework/Node.js/pull/10?email_source=notifications&email_token=AMQ5OOA3FQAV3DGJO7PUAELQXSPYHA5CNFSM4JQ322EKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEGGX3LA#issuecomment-562920876>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMQ5OOARLXYNH5CAGH62ZILQXSPYHANCNFSM4JQ322EA>
> .
>

NesibeInce commented:
Hi @wouterkleijn, @stasel , 
I implemented all the feedback in exercises. I am sorry, i thought I had already implemented the feedback because I immediately opened and observed the change after receiving your feedback.
I think everything is neat and proper now.

Thank you for the compliments, as well.

stasel commented:
Apologies for the late response. Everything looks good now. good job! 👍

NesibeInce commented:
No apologies,  please.
You are already doing great work for me/ us.
Thank you for your time and concern.

Prettige kerstdagen!! 🎄🦌🎅🏻

On Sun, Dec 22, 2019 at 10:37 PM Stasel <notifications@github.com> wrote:

> Apologies for the late response. Everything looks good now. good job! 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HackYourHomework/Node.js/pull/10?email_source=notifications&email_token=AMQ5OOCLVZLISKBLVT6LYCLQZ7MYTA5CNFSM4JQ322EKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHP2JPA#issuecomment-568304828>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMQ5OODENEIPBHRTSTYAOODQZ7MYTANCNFSM4JQ322EA>
> .
>
